src/server/common/helpers/redis-client.js

```diff
@@ -1,63 +1,44 @@
 import IoRedis from 'ioredis'
```

Comment:
We'll want to use the named imports `@returns` refers to by name

```suggestion
import { Cluster, Redis } from 'ioredis'
```